DataFormats/METReco/src/CSCHaloData.cc

```diff
@@ -54,7 +54,7 @@ int CSCHaloData::NumberOfHaloTracks(HaloData::Endcap z) const
   int n = 0 ;
   for(unsigned int i = 0 ; i < TheTrackRefs.size() ; i++ )
     {
-      edm::Ref<reco::TrackCollection> iTrack( TheTrackRefs, i ) ;
+      edm::Ref<reco::TrackCollection> iTrack( TheTrackRefs[i] ) ;
```

Comment:
Say the code was
```
std::vector<Foo*> allFoos;
std::vector<Foo*> someFoos( aFunctionThatMakesChoices(allFoos) );

for(unsigned int i = 0; i< someFoos.size(); i++) {
   std::cout << allFoos[i];
}
```
Would you consider using the wrong size to be a bug? That is exactly what the code was doing.